.github/workflows/tests.yaml

```diff
@@ -189,7 +189,7 @@ jobs:
                   ./scripts/run_in_build_env.sh \
                   "./scripts/tests/run_test_suite.py \
                      --chip-tool ./out/darwin-x64-chip-tool${CHIP_TOOL_VARIANT}-${BUILD_VARIANT}/chip-tool \
-                     --target-skip-glob '{TestGroupMessaging}' \
+                     --target-skip-glob '{TestGroupMessaging,Test_TC_DIAG_TH_NW_1_1}' \
```

Comment:
Is there an issue tracking this?

Reply:
thread is not enabled on Darwin, why we run Thread yaml tests against this platform?

Reply:
Fair point.